Automattic/wp-calypso
# Notifications - enable shortcuts on load regardless of global initialization

Related to # https://github.com/Automattic/loop/issues/165

## Proposed Changes

* Ensures shortcut keys are enabled on load.

To repro the original issue:
* Load /read/notifications, notice shortkeys dont work by default.
* furthermore, if you navigate to other sections of calypso after visiting /read/notifications (without doing something to enable shortkeys like opening and closing a comment), the shortkeys do not work in the bell/panel either.

Notes:
* Notifications uses a global object for some data. When we have 2 notifications instances like we do in calypso this global is shared across both of them. In general this is fine. However, enabling shortkeys in state was only done on initial load if the global was not yet set. In calypso where we have 2 instances of notifications, this means the second instance that loads will not have shortkeys enabled by default. However, shortkeys still get enabled later in user flows as there are other components (such as the nested comment level) that enable shortkeys under other actions.
* Moving this enabling of shorkeys in state out of the conditional seems fine. The code in the conditional in general is for initializing the global variable which this is separate from, the keydown event listeners are added regardless of that condition, and the shortkeys get enabled by interacting with other components regardless of these conditions.

## Testing Instructions



* Load /read/notifications in calypso
* verify shortkeys 'a', 'c', 'j', 'k', etc. start working on initial load. previously you would need to open and close a comment for this to start working.
* navigated out of /read/notifications and verify the notifications bell/panel still works as expected.

Smoke test the standalone app:
* Install this build of the notifications app on your sandbox using the install command in the comment below.
* sandbox widgets and a test site (NOT the public-api).
* visit the front-end of your site and smoke test, there should be no observable changes.


## Pre-merge Checklist
